spec/system/jobseekers/jobseekers_can_add_employments_to_their_job_application_spec.rb

```diff
@@ -40,35 +40,35 @@
     expect(page).to have_content(Date.new(2020, 7, 1).to_formatted_s(:month_year))
   end
 
-  it "displays employment history from newest to oldest job" do
-    visit jobseekers_job_application_build_path(job_application, :employment_history)
-
-    click_on I18n.t("buttons.add_work_history")
-    validates_step_complete(button: I18n.t("buttons.save_employment"))
-
-    fill_in_employment_history(job_title: "Old job")
-
-    click_on I18n.t("buttons.save_employment")
-
-    all(:link, "Add another job").first.click
-
-    fill_in_employment_history(job_title: "Oldest job", start_month: "09", start_year: "2015", end_month: "06", end_year: "2019")
-
-    click_on I18n.t("buttons.save_employment")
+  context "with an employmeent history" do
+    before do
+      build(:employment, reason_for_leaving: nil, job_application: job_application, job_title: "Oldest job",
```

Comment:
this built instance does not seem to be used anywhere

Reply:
It's saved with validate: false - is there a better way of creating an invalid item from a factory?